test/__snapshots__/test.js.snap

```diff
@@ -1112,6 +1112,128 @@ const Impl = require(\\"../implementations/Enum.js\\");
+};
+exports.isImpl = function isImpl(obj) {
+  return utils.isObject(obj) && obj instanceof Impl.implementation;
+};
+exports.convert = function convert(obj, { context = \\"The provided value\\" } = {}) {
+  if (exports.is(obj)) {
+    return utils.implForWrapper(obj);
+  }
+  throw new TypeError(\`\${context} is not of type 'EventTarget'.\`);
+};
+
+exports.create = function create(globalObject, constructorArgs, privateData) {
+  if (globalObject[ctorRegistrySymbol] === undefined) {
+    throw new Error(\\"Internal error: invalid global object\\");
+  }
+
+  const ctor = globalObject[ctorRegistrySymbol][\\"EventTarget\\"];
+  if (ctor === undefined) {
+    throw new Error(\\"Internal error: constructor EventTarget is not installed on the passed global object\\");
+  }
+
+  let obj = Object.create(ctor.prototype);
+  obj = exports.setup(obj, globalObject, constructorArgs, privateData);
+  return obj;
+};
+exports.createImpl = function createImpl(globalObject, constructorArgs, privateData) {
+  const obj = exports.create(globalObject, constructorArgs, privateData);
+  return utils.implForWrapper(obj);
+};
+exports._internalSetup = function _internalSetup(obj, globalObject) {};
+exports.setup = function setup(obj, globalObject, constructorArgs = [], privateData = {}) {
+  privateData.wrapper = obj;
+
+  exports._internalSetup(obj, globalObject);
+  Object.defineProperty(obj, implSymbol, {
+    value: new Impl.implementation(globalObject, constructorArgs, privateData),
+    configurable: true
+  });
+
+  obj[implSymbol][utils.wrapperSymbol] = obj;
+  if (Impl.init) {
+    Impl.init(obj[implSymbol], privateData);
+  }
+  return obj;
+};
+
+exports.install = function install(globalObject) {
+  class EventTarget {
+    constructor() {
+      return exports.setup(Object.create(new.target.prototype), globalObject, undefined);
+    }
+
+    addEventListener(type, callback) {
+      const esValue = this !== null && this !== undefined ? this : globalObject;
+      if (!exports.is(esValue)) {
+        throw new TypeError(\\"Illegal invocation\\");
+      }
+
+      if (arguments.length < 2) {
+        throw new TypeError(
+          \\"Failed to execute 'addEventListener' on 'EventTarget': 2 arguments required, but only \\" +
+            arguments.length +
+            \\" present.\\"
+        );
+      }
+      const args = [];
+      {
+        let curArg = arguments[0];
+        curArg = conversions[\\"DOMString\\"](curArg, {
+          context: \\"Failed to execute 'addEventListener' on 'EventTarget': parameter 1\\"
+        });
+        args.push(curArg);
+      }
+      {
+        let curArg = arguments[1];
+        if (curArg === null || curArg === undefined) {
+          curArg = null;
+        } else {
+          if (!utils.isObject(curArg)) {
+            throw new TypeError(
+              \\"Failed to execute 'addEventListener' on 'EventTarget': parameter 2\\" + \\" is not an object\\"
```

Comment:
I think you could make this all one string by tweaking the source.

Reply:
I'd like to punt on this. This is slightly difficult, considering I'm currently doing

```js
`${errPrefix} + " dictionary"`
```

and `errPrefix` can really be anything, string literal or not. This is also not the only place in the snapshot that has something like this.